src/Storage/GaufretteStorage.php

```diff
@@ -43,7 +43,7 @@ public function __construct(PropertyMappingFactory $factory, FilesystemMapInterf
         $this->protocol = $protocol;
     }
 
-    protected function doUpload(PropertyMapping $mapping, UploadedFile $file, ?string $dir, string $name): void
+    protected function doUpload(PropertyMapping $mapping, File $file, ?string $dir, string $name): void
```

Comment:
Missing check for `UploadedFile|ReplacingFile`

Reply:
Resolving, because this is the same case as the resolved comment.